src/accel/LocalTransporter.cc

```diff
@@ -199,7 +198,7 @@ void LocalTransporter::Push(G4Track const& g4track)
      * back to Geant4.
      */
     track.track_id = TrackId{track_counter_++};
-    track.event_id = event_id_;
+    track.event_id = EventId{0};
```

Comment:
So this is just temporary, and the plan is to set the Celeritas event ID from a global counter and to reset the (single, local) sim data track counter at the start of a new event? And I guess until we do that our track IDs won't be meaningful...

Reply:
Yep, the next PR will include a function to reset these at every event. Track IDs aren't currently very meaningful anyway since they're nondeterministic.